tools/docker-builder/main.go

```diff
@@ -246,13 +246,13 @@ func ConstructBakeFile(a Args) (map[string]string, error) {
 				t.Outputs = []string{"type=registry"}
 			} else if args.Save {
 				n := target
-				if variant != "" {
+				if variant != "" && variant != DefaultVariant { // For default variant, we do not add it.
 					n += "-" + variant
 				}
 
 				tarFiles[n] = ""
 				if variant == PrimaryVariant && hasDoubleDefault {
-					tarFiles[n] = target
+					tarFiles[n] = target + "-" + variant
```

Comment:
We append the tag if doubled up. Using the alias to create another tar file. This line might not be needed or conflict with the alias elsewhere.

Reply:
Looks like this change can be removed.